templates/default.html

```diff
@@ -1,382 +1,32 @@
-        .aqua { color: #00bfbf; }
-        .aqua-background { background-color: #00fafa; }
-        .black { color: black; }
-        .black-background { background-color: black; }
-        .blue { color: #0000bf; }
-        .blue-background { background-color: #0000fa; }
-        .fuchsia { color: #bf00bf; }
-        .fuchsia-background { background-color: #fa00fa; }
-        .gray { color: #606060; }
-        .gray-background { background-color: #7d7d7d; }
-        .green { color: #006000; }
-        .green-background { background-color: #007d00; }
-        .lime { color: #00bf00; }
-        .lime-background { background-color: #00fa00; }
-        .maroon { color: #600000; }
-        .maroon-background { background-color: #7d0000; }
-        .navy { color: #000060; }
-        .navy-background { background-color: #00007d; }
-        .olive { color: #606000; }
-        .olive-background { background-color: #7d7d00; }
-        .purple { color: #600060; }
-        .purple-background { background-color: #7d007d; }
-        .red { color: #bf0000; }
-        .red-background { background-color: #fa0000; }
-        .silver { color: #909090; }
-        .silver-background { background-color: #bcbcbc; }
-        .teal { color: #006060; }
-        .teal-background { background-color: #007d7d; }
-        .white { color: #bfbfbf; }
-        .white-background { background-color: #fafafa; }
-        .yellow { color: #bfbf00; }
-        .yellow-background { background-color: #fafa00; }
-        span.icon > [class^="icon-"], span.icon > [class*=" icon-"] { cursor: default; }
-        .admonitionblock td.icon [class^="icon-"]:before { font-size: 2.5em; text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5); cursor: default; }
-        .admonitionblock td.icon .icon-note:before { content: "\f05a"; color: #005498; color: #003f72; }
-        .admonitionblock td.icon .icon-tip:before { content: "\f0eb"; text-shadow: 1px 1px 2px rgba(155, 155, 0, 0.8); color: #111; }
-        .admonitionblock td.icon .icon-warning:before { content: "\f071"; color: #bf6900; }
-        .admonitionblock td.icon .icon-caution:before { content: "\f06d"; color: #bf3400; }
-        .admonitionblock td.icon .icon-important:before { content: "\f06a"; color: #bf0000; }
-        .conum { display: inline-block; color: white !important; background-color: #222222; -webkit-border-radius: 100px; border-radius: 100px; text-align: center; width: 20px; height: 20px; font-size: 12px; font-weight: bold; line-height: 20px; font-family: Arial, sans-serif; font-style: normal; position: relative; top: -2px; letter-spacing: -1px; }
-        .conum * { color: white !important; }
-        .conum + b { display: none; }
-        .conum:after { content: attr(data-value); }
-        .conum:not([data-value]):empty { display: none; }
-        .literalblock > .content > pre, .listingblock > .content > pre { -webkit-border-radius: 0; border-radius: 0; }
-
+        {{{ style }}}
```

Comment:
> It's to remind my beard

You mean [Beard.js](http://jspopisno1.github.io/Beard/) ? :wink: 

> All variables are HTML-escaped by default. If you want to render unescaped HTML, use the triple mustache: {{{name}}}.

Ok :smile: